src/system-plugins/rov-beacon/beacon.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
```

Comment:
This library is set to use UDB broadcast.  It is currently hardcoded to the 192.168.1.255 address.  This is the only one supported for the 2.x series.  This may need to be changed depending on where we land for the final networking layout with Trident.